src/java/org/apache/cassandra/io/sstable/BloomFilterMetrics.java

```diff
@@ -89,7 +89,7 @@ protected R map(SSTableReader r)
     /**
      * False positive ratio of bloom filter from last read
      */
-    public final GaugeProvider<Double> recentBloomFilterFalseRatio = newGaugeProvider("BloomFilterFalseRatio", readers -> {
+    public final GaugeProvider<Double> recentBloomFilterFalseRatio = newGaugeProvider("RecentBloomFilterFalseRatio", readers -> {
         long falsePositiveCount = 0L;
         long truePositiveCount = 0L;
         long trueNegativeCount = 0L;
```

Comment:
The calls below (`getFalsePositiveCount` etc) should be changed to their recent versions.